From 0135ca620a942030d4ea04787b5f75c983bc7795 Mon Sep 17 00:00:00 2001
Message-Id: <0135ca620a942030d4ea04787b5f75c983bc7795@dist-git>
From: "Daniel P. Berrange" <berrange@redhat.com>
Date: Fri, 26 Sep 2014 12:44:03 +0200
Subject: [PATCH] Fix typo s/EMULATORIN/EMULATORPIN/

Fix the typo in VIR_DOMAIN_TUNABLE_CPU_EMULATORIN

Signed-off-by: Daniel P. Berrange <berrange@redhat.com>
(cherry picked from commit 42571dfa867acb89fea9a731f2e816005c9ab787)

Resolves: https://bugzilla.redhat.com/show_bug.cgi?id=1115898

Signed-off-by: Pavel Hrdina <phrdina@redhat.com>
Signed-off-by: Jiri Denemark <jdenemar@redhat.com>
---
 include/libvirt/libvirt.h.in | 4 ++--
 src/qemu/qemu_driver.c       | 2 +-
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/include/libvirt/libvirt.h.in b/include/libvirt/libvirt.h.in
index 9dbb222..972bdfa 100644
--- a/include/libvirt/libvirt.h.in
+++ b/include/libvirt/libvirt.h.in
@@ -5177,12 +5177,12 @@ typedef void (*virConnectDomainEventDeviceRemovedCallback)(virConnectPtr conn,
 #define VIR_DOMAIN_TUNABLE_CPU_VCPUPIN "cputune.vcpupin%u"
 
 /**
- * VIR_DOMAIN_TUNABLE_CPU_EMULATORIN:
+ * VIR_DOMAIN_TUNABLE_CPU_EMULATORPIN:
  *
  * Macro represents formatted pinning for emulator process,
  * as VIR_TYPED_PARAM_STRING.
  */
-#define VIR_DOMAIN_TUNABLE_CPU_EMULATORIN "cputune.emulatorpin"
+#define VIR_DOMAIN_TUNABLE_CPU_EMULATORPIN "cputune.emulatorpin"
 
 /**
  * VIR_DOMAIN_TUNABLE_CPU_CPU_SHARES:
diff --git a/src/qemu/qemu_driver.c b/src/qemu/qemu_driver.c
index 1b14a12..08f0142 100644
--- a/src/qemu/qemu_driver.c
+++ b/src/qemu/qemu_driver.c
@@ -4876,7 +4876,7 @@ qemuDomainPinEmulator(virDomainPtr dom,
         str = virBitmapFormat(pcpumap);
         if (virTypedParamsAddString(&eventParams, &eventNparams,
                                     &eventMaxparams,
-                                    VIR_DOMAIN_TUNABLE_CPU_EMULATORIN,
+                                    VIR_DOMAIN_TUNABLE_CPU_EMULATORPIN,
                                     str) < 0)
             goto cleanup;
 
-- 
2.1.1

